atd-vze/src/views/Crashes/peopleDataMap.js

```diff
@@ -68,34 +67,32 @@ export const primaryPersonDataMap = [
           },
         },
       },
-      person_type: {
+      prsn_type_lkp: {
         label: "Type",
         editable: false,
-        lookup_desc: "prsn_type_desc",
+        lookup_desc: "label",
       },
       prsn_age: {
         label: "Age",
         editable: true,
         format: "text",
         mutationVariableKey: "personId",
       },
-      gender: {
+      gndr_lkp: {
         label: "Sex",
         editable: true,
-        lookup_desc: "gndr_desc",
+        lookup_desc: "label",
         format: "select",
-        lookupOptions: "atd_txdot__gndr_lkp",
-        lookupPrefix: "gndr",
+        lookupOptions: "lookups_gndr_lkp",
         updateFieldKey: "prsn_gndr_id",
         mutationVariableKey: "personId",
       },
-      ethnicity: {
+      drvr_ethncty_lkp: {
```

Comment:
@johnclary this is the only lookup table i could find for ethnicity. any idea why its prefixed with `drvr`? is this the proper lookup table for the `prsn_ethnicity_id` field?

Reply:
yes! this is one of those that don't match the naming convention we expected. i verified this by checking the CRIS extract `xml` files back when we were researching the lookup tables.